hack/version-and-deploy.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+# version-and-deploy.sh
```

Comment:
there are 2 ways to use git tags.  By default (and the case we're interested in) it will tag the current commit.  On `git push --tags`, this will kick off the CI.  The docker images are handled in the `provider: script \ script: make release` stanza, which is not tag dependent.